### PR TITLE
BAM-2337 updating auction page with the new fields available

### PR DESCRIPTION
**Pull request by PhilWinchester**
Update the auction page with the newest additions to our API!

screen shot 2018-09-20 at 4 39 12 pm
screen shot 2018-09-20 at 4 40 32 pm

### CODE REVIEW

File: docs/auction.rst [hunk 1]
```diff
@@ -206,6 +214,8 @@ Example
               "bamx_product_category_id": 3,
               "name": "Designer Handbag",
               "url": "https://www.amazon.com.example/product/1234",
+              "image_url": "https://www.static.amazon.example/image/path/1234",
+              "stock_status": true,
```
Comment (csdev) [inline docs/auction.rst]: this should be `is_in_stock`

File: docs/auction.rst [hunk 2]
```diff
@@ -123,6 +123,14 @@ the click on the rewritten SmartLink:
      - The raw URL of the product page, for display purposes. (Do not rewrite
        the SmartLink with this URL.)
 
+   * - **image_url**
+     - string
+     - The image URL of the product being sold, for display purposes.
+
+   * - **is_in_stock**
+     - boolean
+     - The stock status of the product being sold.
```
Comment (myfashionhub) [inline docs/auction.rst]: in-stock status?

File: docs/auction.rst [hunk 3]
```diff
@@ -123,6 +123,14 @@ the click on the rewritten SmartLink:
      - The raw URL of the product page, for display purposes. (Do not rewrite
        the SmartLink with this URL.)
 
+   * - **image_url**
+     - string
+     - The image URL of the product being sold, for display purposes.
```
Comment (myfashionhub) [inline docs/auction.rst]: not sure if needed `for display purposes`

File: docs/auction.rst (hunk 1 above)
Comment (PhilWinchester) [reply docs/auction.rst]: indeed it should

File: docs/auction.rst (hunk 2 above)
Comment (PhilWinchester) [reply docs/auction.rst]: following the naming convention

File: docs/auction.rst (hunk 3 above)
Comment (PhilWinchester) [reply docs/auction.rst]: Following the convention used in the product url description

**Pull request merged**